src/WebJobs.Script/Workers/Rpc/WebHostRpcWorkerChannelManager.cs

```diff
@@ -52,26 +51,25 @@ public class WebHostRpcWorkerChannelManager : IWebHostRpcWorkerChannelManager
             _rpcWorkerChannelFactory = rpcWorkerChannelFactory;
             _logger = loggerFactory.CreateLogger<WebHostRpcWorkerChannelManager>();
             _applicationHostOptions = applicationHostOptions;
-            _languageWorkerOptions = languageWorkerOptions;
 
             _shutdownStandbyWorkerChannels = ScheduleShutdownStandbyChannels;
             _shutdownStandbyWorkerChannels = _shutdownStandbyWorkerChannels.Debounce(milliseconds: 5000);
         }
 
-        public Task<IRpcWorkerChannel> InitializeChannelAsync(string runtime)
+        public Task<IRpcWorkerChannel> InitializeChannelAsync(IEnumerable<RpcWorkerConfig> workerConfigs, string runtime)
```

Comment:
This is the major change that caused a ripple throughout the other APIs. Rather than always using the WebHost-level workerConfigs (which could be wrong -- specifically for dotnet-isolated), it now will use the ones passed in.